app/assets/stylesheets/application.css.scss

```diff
@@ -15,6 +15,9 @@
  *= require material
  */
 
+body, #container, .form-background{
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line<br>Opening curly brace `{` should be preceded by one space